src/index.ts

```diff
@@ -3,13 +3,26 @@ import translators from './translators'
 import { findFirstImageSrc } from './utils'
 
 const baseOptions: Partial<NodeHtmlMarkdownOptions> = {
+  /**
+   * Marker used by Slack mrkdwn for bullet lists.
+   */
+  bulletMarker: '•',
+  /**
+   * In Slack mrkdwn: {@link https://api.slack.com/reference/surfaces/formatting#basics}
+   * - Bold text is represented by a single asterisk (*),
+   *   deviating from the standard double asterisks (**) used in regular Markdown.
+   * - Strikethrough is denoted by a single tilde (~),
+   *   in contrast to the standard double tilde (~~) used in regular Markdown.
+   */
+
   strongDelimiter: '*',
+  strikeDelimiter: '~',
```

Comment:
Default delimiter is `~~`
https://github.com/crosstype/node-html-markdown/blob/06a5501523c474f7ae708640d98e4aeabcd67e9b/README.md?plain=1#L157-L161